### PR TITLE
add log message if new tablet doesn't have a sufficiently new timesta…

### PR DESCRIPTION
**Pull request by gentryx**
…mp to be marked up

### CODE REVIEW

File: go/vt/discovery/tablet_stats_cache.go [hunk 1]
```diff
@@ -193,9 +193,13 @@ func (tc *TabletStatsCache) StatsUpdate(ts *TabletStats) {
 
 			// We already have one up server, see if we
 			// need to replace it.
-			if e.healthy[0].TabletExternallyReparentedTimestamp > ts.TabletExternallyReparentedTimestamp {
-				// The notification we just got is older than
-				// the one we had before, discard it.
+			if ts.TabletExternallyReparentedTimestamp < e.healthy[0].TabletExternallyReparentedTimestamp {
+				log.Warningf("not marking healthy master as Up because its externally reparented timestamt is smaller than the highest known timestamp from previous MASTERs: %d < %d ",
```
Comment (alainjobart) [inline go/vt/discovery/tablet_stats_cache.go]: typo: timestamt

Comment (alainjobart) [inline go/vt/discovery/tablet_stats_cache.go]: Also need to add a few imports to fix build.

**Comment by gentryx** [comment]
Ugh, I forgot you'd be seeing this even though CI hasn't completed yet. Sorry about the noise.

 I'll ping you here once the patch is at least compiling. :-)

**Comment by alainjobart** [comment]
LGTM

**Pull request merged**